src/main/java/com/minecolonies/network/messages/AssignmentModeMessage.java

```diff
@@ -0,0 +1,76 @@
+package com.minecolonies.network.messages;
+
+import com.minecolonies.colony.Colony;
+import com.minecolonies.colony.ColonyManager;
+import com.minecolonies.colony.buildings.BuildingFarmer;
+import com.minecolonies.util.BlockPosUtil;
+import io.netty.buffer.ByteBuf;
+import net.minecraft.util.BlockPos;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
+import net.minecraftforge.fml.common.network.simpleimpl.IMessageHandler;
+import net.minecraftforge.fml.common.network.simpleimpl.MessageContext;
+
+/**
+ * Message to change the assignmentMode of the fields of the farmer.
+ */
+public class AssignmentModeMessage implements IMessage, IMessageHandler<AssignmentModeMessage, IMessage>
+{
+
+    private int      colonyId;
+    private BlockPos buildingId;
+    private boolean  assignmentMode;
+
+    /**
+     * Empty standard constructor.
+     */
+    public AssignmentModeMessage()
+    {
+        /**
+         * Intentionally left empty.
+         */
+    }
+
+    /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)